workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -516,27 +517,32 @@ def completed_rundir(configuration, tmpdir_factory):
     else:
         raise NotImplementedError()
 
-    rundir = tmpdir_factory.mktemp("rundir")
+    rundir = tmpdir_factory.mktemp("rundir").join("subdir")
```

Comment:
What's the reason for making the rundir `subdir`?

Reply:
createRun complains about the folder existing if I use an already existing directory.